api/app/resources/users.py

```diff
@@ -0,0 +1,71 @@
+from flask import jsonify, g, abort
+from flask_restful import Resource, reqparse
+
+from ..models import User, db
+# from app import db, auth
+
+
+class Register(Resource):
+    """
+    To Register a new User
+    """
+
+    def __init__(self):
+        """Instantiate route request parameters"""
+        self.parser = reqparse.RequestParser()
+        self.parser.add_argument('username', type=str,
+                                 help='Enter username', location='json')
+        self.parser.add_argument('password', type=str,
+                                 help="User's password",
+                                 location='json')
+        self.parser.add_argument('conf_password', type=str,
+                                 help="enter password second time to confirm",
+                                 location='json')
+        super(Register, self).__init__()
+
+    def post(self):
+        """POST endpoint"""
+        args = self.parser.parse_args()
+        username = args['username']
+        password = args['password']
+        conf_password = args['conf_password']
+
+        if username is None or password is None:
+            abort(400) # missing arguments
```

Comment:
**Code Quality: Critical**

We need to have a more descriptive error message here. Same thing on line 36